src/components/PokeCard.js

```diff
@@ -84,22 +95,24 @@ const PokeCard = ({
           setModalIsOpen(false);
         }}>
         {/* modal content */}
-        <CustomModal
-          genera={generaString}
-          flavorText={flavorText}
-          setIsOpen={setModalIsOpen}
-          name={name}
-          dexNo={dexNo}
-          sprite={sprite}
-          typeTags={typeTags}
-          getPokemonModalAboutContent={getPokemonModalAboutContent}
-          height={height}
-          weight={weight}
-          abilities={abilities}
-          stats={stats}
-          types={types}
-          modalData={modalData}
-        />
+        <animated.div style={modalOpenAnimation}>
+          <CustomModal
+            genera={generaString}
+            flavorText={flavorText}
+            setIsOpen={setModalIsOpen}
+            name={name}
+            dexNo={dexNo}
+            sprite={sprite}
+            typeTags={typeTags}
+            getPokemonModalAboutContent={getPokemonModalAboutContent}
+            height={height}
+            weight={weight}
+            abilities={abilities}
+            stats={stats}
+            types={types}
+            modalData={modalData}
+          />
+        </animated.div>
```

Comment:
i see how you implemented that modal animation, looks straightfoward and came out nice!